ui/components/app/confirm/info/row/__snapshots__/divider.test.tsx.snap

```diff
@@ -0,0 +1,9 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ConfirmInfoRowDivider should match snapshot 1`] = `
+<div>
+  <div
+    style="height: 1px;"
+  />
+</div>
+`;
```

Comment:
not sure why the divider "background-color" doesn't show in the snap. my guess is the tests don't support the design system tokens by default. It appears as expected on the storybook pages